### DISCUSSION TITLE
External converters

### DISCUSSION THREAD

**Discussion by SargonofAssyria** [Koenkk/zigbee2mqtt#12166] [General]
I am just wondering if an external converter for a Tuya device overrules the already present converter in tuya.js

**Comment by Koenkk** [comment]
External converters always overrule definition in tuya.js

**Comment by SargonofAssyria** [comment]
Tnx for the answer. I made an exact copy of the TS011F_plug_3 without the child_lock and named it TS011F_plug_4
But that does not seem to work. Z2M won't even startup.

**Reply by Koenkk** [reply]
> Z2M won't even startup.

Can you provide a error log?

**Reply by SargonofAssyria** [reply]
I enabled debug, add the converter and restart Z2M. But nothing in the log.
Last lines are
info  2022-04-14 17:32:07: Stopped zigbee-herdsman
info  2022-04-14 17:32:07: Stopped Zigbee2MQTT
After that nothing. I see in htop "node index.js" flash but dies immediately.

**Reply by SargonofAssyria** [reply]
This is my external converter.
After removing some typos everything went OK. TNX.
